pages/BasePage.ts

```diff
@@ -0,0 +1,98 @@
+import { Page, Locator } from '@playwright/test';
+
+export class BasePage {
+    readonly page: Page;
+
+    constructor(page: Page) {
+        this.page = page;
+    }
+
+    static readonly PATHS = {
+        LOGIN: '/',
+        DASHBOARD: '/web/index.php/dashboard/index',
+        FORGOT_PASSWORD: '/web/index.php/auth/requestPasswordResetCode',
+        ADMIN: '/web/index.php/admin/viewAdminModule',
+        PIM: '/web/index.php/pim/viewEmployeeList',
+        LEAVE: '/web/index.php/leave/viewLeaveList',
+        TIME: '/web/index.php/time/viewEmployeeTimesheet',
+        RECRUITMENT: '/web/index.php/recruitment/viewCandidates',
+        MY_INFO: '/web/index.php/pim/viewPersonalDetails/empNumber/7',
+        PERFORMANCE: '/web/index.php/performance/searchEvaluatePerformanceReview',
+        DIRECTORY: '/web/index.php/directory/viewDirectory',
+        MAINTENANCE: '/web/index.php/maintenance/purgeEmployee',
+        BUZZ: '/web/index.php/buzz/viewBuzz'
+    } as const;
+
+    static readonly TIMEOUTS = {
+        DEFAULT: 5000,
+        NETWORK: 10000,
+        NAVIGATION: 15000
+    } as const;
```

Comment:
Static should in general should only be used in utility classes. Here it doesn't make sense. These values can either be in a `private` or `protected` state. Or even created as enums